e2e/tests/replica_rebuilding.robot

```diff
@@ -36,3 +36,64 @@ Reboot Replica Node While Replica Rebuilding
         Then Wait until replica on replica node rebuilt
         And Check data is intact
     END
+
+Delete Volume Node While Replica Rebuilding
+    Given Set node-down-pod-deletion-policy to do-nothing
+    And Create deployment with rwo volume
+    And Write 2048 MB data to deployment
+
+    FOR    ${i}    IN RANGE    ${LOOP_COUNT}
+        When Delete replica on volume node to trigger replica rebuilding
+        And During replica rebuilding, delete volume node
+
+        Then Wait for volume attached and unknown
```

Comment:
so the volume will be kept as attached?

Reply:
Yes, this is the current observed behavior.